example/build.gradle

```diff
@@ -0,0 +1,46 @@
+apply plugin: 'com.android.library'
```

Comment:
Could be, I use IntelliJ Community Edition for this since it's not an Android project, so I'm not sure.